docs/CODE-QUALITY.md

```diff
@@ -0,0 +1,24 @@
+| [Code Quality](https://codeclimate.com/github/bcgov/PIMS)              | 3rd party tool to provide insight into the quality or maintainability of the code. **NOTE** - this tool doesn't handle the backend code very well, and as such is not entirely reliable.                                                                                                                                                                            |
+| Unit Tests                                                             | Automated tests to ensure functionality is valid and helps with regression testing.                                                                                                                                                                                                                                                                                 |
+| Functional Testing                                                     | Test Plans are created for each sprint to ensure new enhancements, features and bugs are manually tested.                                                                                                                                                                                                                                                           |
+| Integration Testing                                                    | We have limited integration testing. The API automates an in-memory database to test the Data Access Layer (DAL).                                                                                                                                                                                                                                                   |
+| Automated Functioning Testing                                          | There is currently no automated functional testing.                                                                                                                                                                                                                                                                                                                 |
+| ZAP                                                                    | 3rd party tool to review code security quality. **Not presently hooked up to DevOps pipeline** pipeline.                                                                                                                                                                                                                                                            |
```

Comment:
**Not presently hooked up to DevOps pipeline** pipeline. // <-- remove this word